distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -309,3 +280,74 @@ Here are some helpful suggestions for setting up data backups and recoveries:
 * Run a scheduler program that calls the backup URL on a timed basis.
 * Put indicators in place that can detect when data corruption may have occurred.
 * Testing a backup before recovery is possible. A replicated "staging" Solr server instance can be stood up, and the backup can be copied to that system for testing before moving it to the "production" system.
+
+=== High Availability (Optional)
+
+High Availability in ${branding} is used when a highly available Solr service is needed.
+Solr Cloud is a highly available distributed capability of Solr.
+In Solr Cloud, if a Solr server fails, another Solr server will be able to provide distributed indexing and search capabilities to the ${branding}.
+Solr Cloud enables the ${branding} to be scalable and be fault tolerant.
+The instructions on setting up High Availabilty for ${branding} only include setup in a *NIX environment.
+
+==== Zookeeper
+
+Zookeeper is a distributed hierarchical key-value store, which is used to provide a distributed configuration service, synchronization service, and naming registry for large distributed systems.
+Zookeeper is needed because Solr Cloud uses it to manage its configuration.
+
+===== Prerequisites for Zookeeper
+
+ZooKeeper 3.4.5
+
+*NIX environment
+
+JDK 8 or greater
+
+===== Installing Zookeeper
+
+Refer to https://zookeeper.apache.org/doc/r3.1.2/zookeeperStarted.html#sc_Download for installation instructions.
+
+[NOTE]
+====
+Minimum three Zookeeper nodes required.
+====
+
+==== Solr Cloud
+
+Solr Cloud provides distributed capabilities needed by Solr servers to communicate and establish a highly available, fault tolerant cluster of Solr instances.
+
+===== Prerequisites for Solr Cloud
+
+Solr 6
+
+*NIX environment
+
+Zookeeper
+
+JDK 8 or greater
+
+====== Installing Solr Cloud
+
+Repeat the following procedure for each node that will be part of the Solr Cloud cluster:
+
+. Refer to https://cwiki.apache.org/confluence/display/solr/Taking+Solr+to+Production for installation instructions.
```

Comment:
Started going through these instructions thoroughly for the hero and found that this link points to a guide that does not work on Unix machines. Talked to @figliold and decided that we would instead just link them to the home directory of the Solr guide. Here's the link: https://cwiki.apache.org/confluence/display/solr/Apache+Solr+Reference+Guide